Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -310,6 +310,17 @@ theorem tendsto_nhds_of_cauchySeq_of_subseq [Preorder β] {u : β → α} (hu :
   le_nhds_of_cauchy_adhp hu (mapClusterPt_of_comp hf ha)
 #align tendsto_nhds_of_cauchy_seq_of_subseq tendsto_nhds_of_cauchySeq_of_subseq
 
+/-- Any shift of a Cauchy sequence is also a Cauchy sequence. -/
+theorem cauchySeq_shift {u : ℕ → α} (k : ℕ) : CauchySeq u ↔ CauchySeq fun n ↦ u (n + k) := by
+  constructor <;> intro h
+  · exact h.comp_tendsto (tendsto_add_atTop_nat k)
+  · rw [cauchySeq_iff] at h ⊢
+    intro V mV
+    obtain ⟨N, h⟩ := h V mV
+    use N + k
+    intro a _ b _
+    convert h (a - k) (by omega) (b - k) (by omega) <;> omega
```

Comment:
```suggestion
    intro a ha b hb
    convert h (a - k) (Nat.le_sub_of_add_le ha) (b - k) (Nat.le_sub_of_add_le hb) <;> omega
```
`omega` is a bit overkill for the first two side goals.

Reply:
Done.